autoscraper/auto_scraper.py

```diff
@@ -1,6 +1,8 @@
-import json
 import os
+import json
+from functools import reduce
 from urllib.parse import urljoin, urlparse
```

Comment:
Unused import.
Also about the order of imports: https://www.python.org/dev/peps/pep-0008/#imports